_includes/manuals/1.8/3.2.3_installation_scenarios.md

```diff
@@ -142,7 +142,10 @@ foreman-installer \
   --foreman-proxy-oauth-consumer-secret=<secret here>
 {% endhighlight %}
 
-Ensure the dns-interface argument is updated with the correct network interface name for the DNS server to listen on.
+Ensure the dns-interface argument is updated with the correct network
+interface name for the DNS server to listen on. After configuration, make
+sure to create Subnet in the Foreman interface for the particular Smart Proxy
```

Comment:
`create a Subnet` and maybe say, `in Foreman under *Infrastructure > Subnets* for the`